UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -28,8 +28,12 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double JoulesInOneJoule => 1;
 
+        protected override double KilobritishThermalUnitsInOneJoule => 9.4781339449889105832843629746176e-7;
```

Comment:
I forgot to verify the latest units, I see a difference here with google (BTU, not kBTU): `9.47817...e-7`
https://www.google.no/search?q=britishThermalUnitsInOneJoule&oq=britishThermalUnitsInOneJoule&aqs=chrome..69i57.224j0j7&sourceid=chrome&ie=UTF-8